### PR TITLE
DPCS alpha server

### PR DESCRIPTION
**Pull request by patrikos94**
Server for executing simple operations, like adding, deleting and changing crash reports, crash groups and solutions to crash groups.
Using Django 1.8.
Supports basic HTTP request(PUT, POST, GET, DELETE).
REST API made with Django Rest Framework.

### CODE REVIEW

**Comment by bdfhjk** [comment]
Your code seems nice. I'll try to review it carefully tomorrow. My first feedback is: should we put all files inside the /server/alpha/django/... location instead of alpha_server/server_django? I think it will look nicer, if we later add /server/alpha/flask and possible /server/juju/

**Comment by patrikos94** [comment]
Yes, I think you're right. It'll be much better if we throw it inside server/alpha to keep the project clean.

File: alpha_server/server_django/alpha/models.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+from django.db import models
+
+
+
+class CrashGroup(models.Model):
+    crash_group_id = models.IntegerField(primary_key=True)
+    def __str__ (self):
+        return str(self.crash_group_id)
+
```
Comment (hubertta) [inline alpha_server/server_django/alpha/models.py:9]: I suggest to make the source code PEP8-compliant.

File: alpha_server/server_django/alpha/views.py [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+            report = CrashReport.objects.get(pk=added)
+        except CrashReport.DoesNotExist:
+            return Response("Could not add the crash report ... Make sure you passed correct arguments ...",
+                status=status.HTTP_400_BAD_REQUEST)
+
+        serializer = CrashReportSerializer(report)
+        return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+
+        #serializer = CrashReportSerializer(data=request.data)
```
Comment (hubertta) [inline alpha_server/server_django/alpha/views.py:157]: ??

**Comment by hubertta** [comment]
Nice work ;] please remove the sqlite3 db file (and include fixtures if necessary).

**Comment by bdfhjk** [comment]
@patrikos94  Please adjust to previous comment, I'll review it actively and merge if everything will work fine! :+1:  Good job! :smile:

**Comment by bdfhjk** [comment]
The code looks promising, but there are a few problems so far. Feedback from active review:
1. There is no /vd1/paths service.
2. There is no easy way to assign solution to bucket. Displaying them in django-admin is not convenient to ex. find the bucket that's largest non-assigned. That was the most important feature.
3. Buckets in django admin are displayed like a big list : "10345". Putting a small description next to each + hit count + assignment status, along with the possibility to sort / hide depending on parameters will be really helpful. 

Anyway it's a good start! Thanks!

File: alpha_server/server_django/alpha/views.py (hunk 2 above)
Comment (patrikos94) [inline alpha_server/server_django/alpha/views.py:157]: alpha_server/server_django/... is an older commit

**Comment by patrikos94** [comment]
2,3) Django admin is not meant for alpha testers. The users can add their soution/crash reports using POST requests. Solutions can be changed via PUT request to vd1/solutions/<id of a solution>

**Pull request merged**